src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -54,7 +54,7 @@ public void Configure(OpenIddictServerOptions options)
                 return;
             }
 
-            options.IgnoreScopePermissions = true;
+            options.IgnoreScopePermissions = false;
```

Comment:
`false` is the default value, so we can remove this line.

Reply:
Removed the line